backend/Controllers/UserController.cs

```diff
@@ -0,0 +1,66 @@
+using Microsoft.AspNetCore.Mvc;
+using System;
+using WhaleSpotting.Services;
+using WhaleSpotting.Helpers;
+using WhaleSpotting.Models.Request;
+using WhaleSpotting.Models.Database;
+using WhaleSpotting.Models.Response;
+
+namespace WhaleSpotting.Controllers 
+{
+    [ApiController]
+    [Route("/users")]
+    public class UserController : ControllerBase 
+    {
+        private readonly IAuthService _authService;
+        private readonly IUserService _userService;
+
+        public UserController
+        (
+            IAuthService authService,
+            IUserService userService
+        ) {
+            _authService = authService;
+            _userService = userService;
+        }
+
+        [HttpPost("create")]
+        public ActionResult<CreateUserResponse> Create([FromHeader] string authorization, [FromBody] CreateUserRequest newUserRequest)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
+            }
+
+            if (authorization is null)
+            {
+                return new UnauthorizedResult();
+            }
+
+            try
+            {
+                var encodedUsernamePassword = authorization.Substring("Basic ".Length).Trim();
+                string usernamePassword = AuthHelper.Base64Decode(encodedUsernamePassword);
+                int separatorIndex = usernamePassword.IndexOf(':');
+
+                var splitUsernamePassword = usernamePassword.Split(':');
+                var username = splitUsernamePassword[0];
+                var password = splitUsernamePassword[1];
+
+                var check = _authService.IsValidLoginInfo(username, password);
+                if (!check)
+                {
+                    return new UnauthorizedResult();
+                }
+                
+                User createdUser = _userService.Create(newUserRequest);
+
+                return new CreateUserResponse(newUserRequest, true);  
```

Comment:
As per below, just return the created `User`:
```suggestion
                return createdUser;
```
You hard-code `true` for the success value, but you'd never have that value as `false`! By returning the created `User` we imply success (it's also implicit in the success response of the HTTP request).

In fact, you should actually be returning 201 Created:
```suggestion
                return Created("/api", createdUser);
```
The `"/api"` is a bit of a hack, since none of the `Created` methods allow us to return the response without giving "the URL this newly-created resource can be found at", but we don't have a "get user" endpoint so there's no such URL!